test/integration/patients/patient/patient-flow.js

```diff
@@ -389,6 +404,37 @@ context('patient flow page', function() {
       .find('.comment__item')
       .should('have.length', 3);
 
+    cy.sendWs({
+      category: 'AttachmentAdded',
+      resource: {
+        type: 'patient-actions',
+        id: testFlowAction.id,
+      },
+      payload: {
+        clinician: {
+          type: 'clinicians',
+          id: '95014c50-2401-4d95-83e1-4a39fb928c90',
+        },
+        file: {
+          type: 'files',
+          id: '70294032-539b-4dc0-93df-9e4ff2eee8c1',
+        },
+        attributes: {
+          path: `patients/${ testPatient.id }/HRA.pdf`,
+          view: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/view/HRA.pdf`,
+          download: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/download/HRA.pdf`,
+        },
+      },
+    });
```

Comment:
The `attributes` object here isn't currently being included in what the BE sends. They're only sending `payload.clinician` and `payload.file`.

BE example message structure is [here](https://github.com/RoundingWell/care-ops-backend/blob/develop/tests/Notifications/Data/Messages/attachment-added-message-structure.json).

If we're given the `path`, `view`, and `download` attributes, we should be able to construct an attachment model ourselves and add it to the UI. We'd need the same data for the `FileReplaced` notification as well.

Otherwise we currently only have the file `id`. Meaning we'd have to make a request to the BE api to get the file model.